app/controllers/spotlight/featured_images_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Spotlight
+  # Handles requests to upload images for exhibit thumbnails
+  class FeaturedImagesController < Spotlight::ApplicationController
+    load_and_authorize_resource instance_name: :featured_image
+
+    def create
+      @featured_image.save!
```

Comment:
Are we relying on the exception for anything, or could this be written as something like: `if @featured_image.save && @featured_image.image.file`? (maybe with `@featured_image.image.file` behind a clearly named accessor?)